We are currently treating this as a real library and sonaming it appropriately
--- a/csharp/qyoto/CMakeLists.txt
+++ b/csharp/qyoto/CMakeLists.txt
@@ -42,6 +42,7 @@
 LINK_DIRECTORIES (${LIBRARY_OUTPUT_PATH})
 
 ADD_LIBRARY (qyotoshared SHARED ${SRC_QYOTO_SHARED})
+set_target_properties(qyotoshared PROPERTIES VERSION 1d.0.0 SOVERSION 1d)
 TARGET_LINK_LIBRARIES (qyotoshared smokeqt ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY})
 
 ADD_LIBRARY (qyoto MODULE ${SRC_QYOTO})
